### ISSUE TITLE
Psionic Bug Batch

### ISSUE THREAD

**Issue by Ceremorph** [AgonyAunt/Sigil-City-of-Doors-PW#314]
From Strawberry Jam:
- Firing certain powers as Psionic still doesn't break invisibility. Dismissal, crystallize, hail of crystal, shower of crystal, disintegrate, assimilate, ...
- The psionic power Assimilate is "target: living creature", still works on undead and constructs
- The psion discipline feats Metacreativity and Telepathy still refer incorrectly, at the end of the descriptions "At 9th level the Nomad gets ..." for Metacreativity (should be Shaper, and Telepath is "At 9th level the Seer gers" should be Telepath
- For Psion bar the only thing that doesn't work now is if you select the metapsionics or the augment from the hotbar, it doesn't light up on the Psibar GUI. It only lights up on the GUI if you select it from the GUI. Not sure if it can be fixed to work both ways.
- The Psion power Cheat Fate doesn't have an icon. 
- lso forgot about the Psion power Psychic Surgery. It still strips all effects including beneficial effects and spells from a target instead of just the negative ones.

**Comment by Clangeddin** [comment]
"For Psion bar the only thing that doesn't work now is if you select the metapsionics or the augment from the hotbar, it doesn't light up on the Psibar GUI. It only lights up on the GUI if you select it from the GUI. Not sure if it can be fixed to work both ways."

This one is not a bug, it's intended to work this way. Simply clicking twice on the icon will "fix" this problem. This also happens only if you switch between hotbar and gui, if one sticks always  to the GUI this problem never arises.

**Comment by Xndar** [comment]
-There is nothing in the code of Psychic Surgery that indicates why it would do anything of the sort. All it does is copied directly from the Restoration spell script, with a few things added. If someone wants to take a look/confirm this bug IG, I'd appreciate it. ((Note, I made some changes to the original script a few updates back...so this may be an old bug unless its been confirmed recently))

-I submitted the icon for Cheat Fate quite a while ago, so I'm not sure why it never made it in.

-This week I'll work on removing verbal components and fixing the 'hostile' flag on these (and other) powers.
